### DISCUSSION TITLE
Is it possible to close a window and leave a placeholder in its place?

### DISCUSSION THREAD

**Discussion by Asday** [i3/i3#5486] [Q&A]
Currently I launch a desktop twitter client and resize it so it's skinnier, which takes some amount of time, but for the sake of discussion we could pretend I do it in the config with a saved layout and `exec`.

Unfortunately, this twitter client doesn't play well with i3 and crashes every so often.  I then need to `xkill`/`pkill` it, relaunch it, move it to the right container, and resize it again.

Is there some way I could configure i3 to, when it sees that window close, leave a placeholder with an appropriate swallow in its place?

**Comment by okraits** [comment]
There is the _open_ command which spawns a placeholder window. This is not officially supported, though.

**Reply by Asday** [reply]
I ran `i3-msg open` and it certainly did something, but I'm a little too Sunday-brained to figure out how I'd get it to replace a closed window.  Were I to wrap the application in a script that runs `i3-msg open` after it quits, that wouldn't keep the layout, and I don't think it's guaranteed to open in the same workspace even, right?

Certainly an interesting thing to investigate though.
